app/cmd/set.go

```diff
@@ -4,40 +4,60 @@ import (
 	"fmt"
 	"os"
 
+	appConfig "github.com/gSchool/glearn-cli/app/config"
 	"github.com/spf13/cobra"
-	"github.com/spf13/viper"
 )
 
+// UpgradeConfig is an initialized Boolean indicating that the user wants to
+// upgrade their config
+var UpgradeConfig bool
+
+// APIToken is an initialized string used for holding it's flag value
+var APIToken string
+
+func NewSetCommand() *cobra.Command {
+	setCmd.Flags().StringVarP(&APIToken, "api_token", "", "", "Your Learn api token")
+	setCmd.Flags().BoolVarP(&UpgradeConfig, "upgrade", "", false, "Upgrade your CLI config file")
+	return setCmd
+}
+
 var setCmd = &cobra.Command{
-	Use:   "set --api_token=value",
-	Short: "Set your your credentials for ~/.glearn-config.yaml",
-	Long: `
-In order to use learn resources through our CLI you must set your
-credentials inside ~/.glearn-config.yaml
-	`,
+	Use:   "set",
+	Short: fmt.Sprintf("Set your your credentials in %s", appConfig.ConfigPath()),
```

Comment:
remove extra "your"
```suggestion
	Short: fmt.Sprintf("Set your credentials in %s", appConfig.ConfigPath()),
```

Reply:
Addressed in cd63bdc. Thanks for seeing that!